pages/docs/index.js

```diff
@@ -279,6 +279,21 @@ export default function DocsIndex() {
           content="Discover more people in your LinkFree community"
         />
         <link rel="icon" href="/favicon.ico" />
+        <style>
+          {`
```

Comment:
the project uses tailwind can't it be done by using tailwind classes itself

Reply:
> the project uses tailwind can't it be done by using tailwind classes itself

I will check tomorrow and try to work on it